File: obg.Domain/Entities/Administrator.cs
using obg.Domain.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace obg.Domain.Entities
{
    public class Administrator : User
    {
        public Pharmacy Pharmacy { get; set; }

        public Administrator() { }
        public Administrator(string name, int code, string email, string password, string address, RoleUser role, string registerDate) : base (name, code, email, password, address, role, registerDate)
        {
            Pharmacy = new Pharmacy();
        }

    }
}

File: ObgApp/src/app/models/administrator.ts
import { Pharmacy } from "./pharmacy";
import { RoleUser, User } from "./user";

export class Administrator extends User {

    pharmacies: Pharmacy[] | null;

    constructor(name: string | null, code: number | null, email: string | null, password: string | null, address: string | null, role: RoleUser | null, registerDate: string | null, pharmacies: Pharmacy[] | null) {
        super(name, code, email, password, address, role, registerDate);
        this.pharmacies = pharmacies;
    }
}